tensorflow_encrypted/protocol/pond.py

```diff
@@ -426,6 +426,24 @@ def sum(self, x, axis=None, keepdims=None):
     def reduce_sum(self, x, axis=None, keepdims=None):
         return self.sum(x, axis, keepdims)
 
+    @memoize
+    def where(self, x):
+        with tf.name_scope('where'):
+            x_on_0, x_on_1 = x.unwrapped
+
+            with tf.device(self.server_0.device_name):
+                z_on_0 = tf.where(x_on_0.value)
+
+            with tf.device(self.server_1.device_name):
+                z_on_1 = tf.where(x_on_1.value)
+
+            return PondPublicTensor(self, self.tensor_factory.Tensor.from_native(z_on_0), self.tensor_factory.Tensor.from_native(z_on_1), x.is_scaled)
```

Comment:
see comments for `_equal_public_public` -- you'll need to forward calls to `where` which should also take care of calling `from_native`